NSS-Day-Cohort-53/tabloid-full-stack-amicably-profound-insects
# Bw view published posts

#54 View All Published Posts

Added file Post.cs to Models directory.

Added file IPostRepository.cs and PostRepository.cs to Repository directory.

Added file PostController.cs to Controller Repository.

Added IPostRepository/PostRepository to Startup.cs.

Added file postManager.js to modules directory.

Added PostList.js to components directory.

Added a post route path to ApplicationViews.js
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bw-ViewPublishedPosts

    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.

When user selects the Posts menu option
Then they should be directed to the Posts list page
showing the title, author and category. It should ONLY contain approved Posts
It should ONLY contain Posts with a publication date that is in the past
It should be in order of publication date with the most recent on top
 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
